scaper/core.py

```diff
@@ -214,7 +215,7 @@ def generate_from_jams(jams_infile,
 
     # Cast ann.sandbox.scaper to a Sandbox object
     ann.sandbox.scaper = jams.Sandbox(**ann.sandbox.scaper)
-    soundscape_audio, event_audio_list = \
+    soundscape_audio, event_audio_list, scale_factor = \
```

Comment:
This returns `scale_factor`, but we're not saving the `fix_clipping` and `peak_normalize` vals anywhere right? The logic being that `ref_db` was updated when this JAM was created, I think..

Hmm, what if someone does `sc.generate` with `no_audio=True`, but with `peak_normalize=True` or `fix_clipping=True`. We may have to store it in the JAMS after all.

Reply:
we're now storing these in the jams